ui/src/variables/actions/thunks.ts

```diff
@@ -433,7 +434,10 @@ export const selectValue = (variableID: string, selected: string) => async (
 
   await dispatch(selectValueInState(contextID, variableID, selected))
   // only hydrate the changedVariable
-  dispatch(hydrateVariables(true))
-  // dispatch(hydrateChangedVariable(variableID))
+  if (isFlagEnabled('hydratevars')) {
```

Comment:
this is the most badass thing i've ever seen

Reply:
pushing the limits of feature flagging:

https://media.giphy.com/media/WsvLlmmjx9tnmeTPNc/giphy.gif